source/JSONWebToken-Tests.package/JSONWebTokenTest.class/instance/testRoundtrip.st

```diff
@@ -3,9 +3,9 @@ testRoundtrip
 	| jws tokenString materialized |
 	
 	jws := JsonWebSignature new
-		algorithmName: 'HS256';
-		payload: (JWTClaimsSet new
-			at: 'bar' put: 'foo').
+				algorithmName: 'HS256';
```

Comment:
What is that kind of indentation. I'm not a fan of it

Reply:
ignore it - just format the method with regular tools